### DISCUSSION TITLE
关于baby-plonk代码运行遇到的问题

### DISCUSSION THREAD

**Discussion by 0xxEric** [coset-io/zkp-academy#23] [Q&A] [answered]
运行baby-plonk时，无法进入pairing部分，（assert b.pairing(self.X_2, W_quot) ==b.pairing(b.G2, ec_comb)）
目前来看，应该是无法运行pairing函数。（pairing由外部的py_ecc库中的bn128引入）
目前来看，很可能不是代码本身的问题，可能是VScode运行，未能成功运行外部库的问题？
报错反馈如下：
<img width="750" alt="a6db1383bc71c1639bea00b2431fa4d" src="https://github.com/Antalpha-Labs/zkp-academy/assets/171760626/af063fda-287c-4a76-a2f9-726ae588be8f">

**Comment by luckyyang** [comment, marked as answer]
确认了一下, 是 windows 环境, 这个代码似乎 windows 下面跑不了, unix 是可以跑的
